fix/empty-wheelbarrows.lua

```diff
@@ -0,0 +1,21 @@
+--checks all wheelbarrows on map for rocks stuck in them. If a wheelbarrow isn't in use for a job (hauling) then there should be no rocks in them
+--rocks will occasionally get stuck in wheelbarrows, and accumulate if the wheelbarrow gets used.
+--this script empties all wheelbarrows which have rocks stuck in them.
+
+
+for _,e in ipairs(df.global.world.items.other.TOOL) do
+    -- wheelbarrow must be on ground and not in a job
+    if ((not e.flags.in_job) and e.flags.on_ground) then
+        if e.subtype.id == "ITEM_TOOL_WHEELBARROW" then
+            local items = dfhack.items.getContainedItems(e)
+            print('Emptying wheelbarrow: ' .. dfhack.items.getDescription(e, 0))
+            if #items > 0 then
+                print('Emptying wheelbarrow: ' .. dfhack.items.getDescription(e, 0))
```

Comment:
prints on L11 and L13 are the same, is this intended?

also, having a quiet option would be beneficial for people who might want to run this job automatically via repeat, so an option to suppress nonerror output would likely be desirable

Reply:
Nope, the first print is a mistake on my part. I'll add quiet options too

Reply:
That's a mistake, L11 shouldn't be there. I'll implement quiet options as well as well as a dry run option.